packages/nimble-components/src/menu/tests/menu.mdx

```diff
@@ -0,0 +1,33 @@
+import {
+    Controls,
+    DocsStory,
+    Meta,
+    Stories,
+    Title,
+    Description
+} from '@storybook/blocks';
+import TargetDocs from '../../patterns/anchor/tests/target-docs.mdx';
+import * as menuStories from './menu.stories';
+
+<Meta of={menuStories} />
+
+<Title of={menuStories} />
+<Description of={menuStories} />
+
+<DocsStory of={menuStories.menu} expanded={false} />
+<Controls of={menuStories.menu} />
+
+# Usage Docs
+
+## Child Elements
+
+-   `nimble-menu-item` - Use this child element to execute a function when the item is activated or to include a sub-menu. To include a sub-menu, place a `nimble-menu` as a child of a `nimble-menu-item`.
+-   `nimble-anchor-menu-item` - Use this child element to navigate to a URL when the item is activated.
+
+### `nimble-anchor-menu-item` Target Configuration
+
+<TargetDocs />
+
+## Custom Header
+
+The menu can be configured to display a custom header with arbitrary content. This could include other nimble components, or even custom elements.
```

Comment:
I think we may want to render the customMenu story below this description so someone can see the show code example